src/components/SidePanels/DetailsPanel/RegionItem.tsx

```diff
@@ -130,11 +129,6 @@ const RegionAction: FC<any> = observer(({
           look='alt'
           style={{ width: 36, height: 32 }}
         />
-        <RegionActionButton
-          icon={region.hidden ? <IconEyeClosed/> : <IconEyeOpened/>}
-          onClick={region.toggleHidden}
-          hotkey="region:visibility"
-        />
```

Comment:
Are we removing the ability to toggle the visibility through the button? I'm not sure that was stated in the ticket, seems like a regression?

Reply:
Good catch, thank you. I removed to test, and I forget to add back, I should just remove the hotkey. Thanks!